### PR TITLE
adding assertion for javascript single value

### PR DESCRIPTION
**Pull request by awesomeousPrime**
Adding an assertion to check for a single javascript value.

### CODE REVIEW

File: src/Behat/FlexibleMink/Context/JavaScriptContext.php [hunk 1]
```diff
@@ -103,4 +103,24 @@ protected function getRawOrJson($value)
...
+
+    /**
+     * {@inheritdoc}
+     *
+     * @Then the javascript variable :variableName should have the value of :expectedValue
+     */
+    public function assertJavascriptVariable($variableName, $expectedValue)
+    {
+        $returnedJsonData = $this->getSession()->evaluateScript(
+            'return JSON.stringify(' . $variableName . ');'
```
Comment (RedEnchilada) [inline src/Behat/FlexibleMink/Context/JavaScriptContext.php]: Is there a reason to JSON-encode the variable and then decode it after returning? You should be able to just return the variable as-is, and the driver should handle returning the proper variable type.

**Comment by RedEnchilada** [comment]
@awesomeousPrime Code review complete. If I'm wrong about the type handling and it does need to be handled that way, then this is good to merge as-is.

File: src/Behat/FlexibleMink/Context/JavaScriptContext.php (hunk 1 above)
Comment (awesomeousPrime) [inline src/Behat/FlexibleMink/Context/JavaScriptContext.php]: Yes this is not needed.  I've gone ahead and changed this.

**Comment by awesomeousPrime** [comment]
@RedEnchilada code review change has been made.

**Comment by RedEnchilada** [comment]
👍

**Pull request merged**